src/DependencyInjection/Configuration.php

```diff
@@ -22,8 +22,9 @@ final class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('integralservice_sylius_paybox');
+        $rootName = 'integralservice_sylius_paybox';
+        $treeBuilder = new TreeBuilder($rootName);
+        $rootNode = method_exists($treeBuilder, 'getRootNode') ? $treeBuilder->getRootNode() : $treeBuilder->root($rootName);
```

Comment:
It seems that calling $treeBuilder->getRootNode() is useless in case the method does exist. I think it could be better to inverse your condition and only call root when its needed (sf4 compatibility) like :
`$rootNode = !method_exists($treeBuilder, 'getRootNode') ?? $treeBuilder->root($rootName);`

Reply:
Hi,
It seems it will be possible to configure options by this configuration in the future (TODO in code). So this variable should be used later.

Reply:
Alright then it's ok ! thanks